src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -261,79 +351,137 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
         return decoded;
     }
 
+};
+
+
+class ov::genai::VLMPipeline::VLMContinuousBatchingAdapter : public ov::genai::VLMPipeline::VLMPipelineBase {
+public:
+    ContinuousBatchingPipeline m_impl;
+
+    VLMContinuousBatchingAdapter(
+        const std::filesystem::path& models_dir,
+        const SchedulerConfig& scheduler_config,
+        const std::string& device,
+        const ov::AnyMap& properties
+    ): VLMPipelineBase{models_dir, device, properties}, m_impl{
+        models_dir / "openvino_language_model.xml", 
```

Comment:
I suppose we can pass `models_dir`

And CB pipeline can understand, that `openvino_model.xml` is not available and try `openvino_language_model.xml` then. Also, CB impl need to create embedding model and use it in `ModelRunner` to compute embeddings for generated tokens

Reply:
Done.